processing/site_info.py

```diff
@@ -19,29 +19,27 @@ def __init__(self, SITE_ID):
             config.read_file(cfg)
             if config.has_section(cfg_section):
                 self.site_info_ws = config.get(cfg_section, 'site_info')
-                self._load_site_dict(SITE_ID)
+                self._load_site_dict(site_id)
             else:
                 self.site_info_ws = None
                 _log.error('Cannot find web service from config.')
 
-    def _load_site_dict(self, SITE_ID):
-        url = self.site_info_ws + SITE_ID
+    def _load_site_dict(self, site_id):
+        url = self.site_info_ws.format(site_id=site_id)
         content = self.ws_util.get_content(url)
 
         resp = json.loads(content)
-        grp_header = list(resp['values']['GRP_HEADER'].values())[0]
+        grp_header = list(resp['HEADER'].values())[0]

```

Comment:
Probably good to change this to a `get(...)` operation a well for consistency, but I will approve as is.